src_c/doc/transform_doc.h

```diff
@@ -14,6 +14,7 @@
 #define DOC_PYGAMETRANSFORMLAPLACIAN "laplacian(surface, dest_surface=None) -> Surface\nfind edges in a surface"
 #define DOC_PYGAMETRANSFORMAVERAGESURFACES "average_surfaces(surfaces, dest_surface=None, palette_colors=1) -> Surface\nfind the average surface from many surfaces."
 #define DOC_PYGAMETRANSFORMAVERAGECOLOR "average_color(surface, rect=None, consider_alpha=False) -> Color\nfinds the average color of a surface"
+#define DOC_PYGAMETRANSFORMINVERT "average_color(surface, dest_surface=None) -> Surface\ninverts the RGB elements of a surface"
```

Comment:
```suggestion
#define DOC_PYGAMETRANSFORMINVERT "invert(surface, dest_surface=None) -> Surface\ninverts the RGB elements of a surface"
```

Docs are slightly wrong here.